yt_dlp/extractor/prankcast.py

```diff
@@ -0,0 +1,77 @@
+from datetime import datetime
+from .common import InfoExtractor
+from ..utils import parse_duration
+
+
+class PrankCastIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?prankcast\.com/.*/showreel/(?P<id>[0-9]+)'
+    _TESTS = [{
+        'url': 'https://prankcast.com/Devonanustart/showreel/1561-Beverly-is-back-like-a-heart-attack-',
+        'info_dict': {
+            'id': '1561',
+            'ext': 'mp3',
+            'title': 'Beverly is back like a heart attack!',
+            'uploader': 'Devonanustart',
+            'upload_date': '20220825'
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        # Extract the JSON
+        json_info = self._search_nextjs_data(webpage, video_id)['props']['pageProps']['ssr_data_showreel']
+
+        # Get the broadcast URL and the recording hash.
+        # The full URL is {broadcast_url}/{recording_hash}.mp3
+        broadcast_url = json_info.get('broadcast_url')
+        recording_hash = json_info.get('recording_hash')
+        url = broadcast_url + recording_hash + ".mp3"
+
+        # Get broadcast title
+        broadcast_title = json_info.get('broadcast_title') or self._og_search_title(webpage)
+
+        # Get dates
+        start_date = json_info.get('start_date')
+        if 'start_date' != '':
+            start_date = start_date.replace('Z', '')
+
+        end_date = json_info.get('end_date')
+        if 'end_date' != '':
+            end_date = end_date.replace('Z', '')
```

Comment:
Please correct me if i'm wrong, i think `start_date` and `end_date` can parsed with `parse_iso8601` from `utils`

Reply:
Yeah, you're right. I'll change these to use parse_iso8601